src/rules/derived-has-same-inputs-outputs.ts

```diff
@@ -0,0 +1,110 @@
+import type * as ESTree from "estree"
+import { createRule } from "../utils"
+import type { RuleContext } from "../types"
+import { extractStoreReferences } from "./reference-helpers/svelte-store"
+
+export default createRule("derived-has-same-inputs-outputs", {
+  meta: {
+    docs: {
+      description: "",
+      category: "Best Practices",
```

Comment:
Perhaps `"Stylistic Issues"` is a more appropriate category.

```suggestion
      category: "Stylistic Issues",
```

Reply:
updated! https://github.com/ota-meshi/eslint-plugin-svelte/commit/ba6dcccb7f8c182895f8084b2f951c4087aeaaa1
`conflictWithPrettier` should be `false` right?🤔